### DISCUSSION TITLE
Remove Git LFS

### DISCUSSION THREAD

**Discussion by TysonMN** [RehanSaeed/Serilog.Exceptions#242] [Q&A]
This repository is configured to support large binary files via Git LFS in [its `.gitattributes`](https://github.com/RehanSaeed/Serilog.Exceptions/blob/master/.gitattributes#L17-L44).  However, the only files with any of those extension are [these three png files](https://github.com/RehanSaeed/Serilog.Exceptions/tree/master/Images).  They don't seem large to me, and they don't seem to be changing very often.  (Two of them have never changed and one has changed once.)

I was considering contributing some improvements to this repository, but the use of Git LFS is making this more difficult for me.  Instead of troubleshooting the issue I had, I think a better solution would be to remove the use of Git LFS.

Can [these lines in the `.gitattributes` file](https://github.com/RehanSaeed/Serilog.Exceptions/blob/master/.gitattributes#L17-L44) be removed?

I could create a PR with this change.

**Comment by RehanSaeed** [comment]
Hey @TysonMN!

I'm sorry to hear you're having trouble. I usually add a `.gitattributes` by default these days as if you want to do it later, moving to git LFS means rewriting history. In theory, as long as you have LFS installed, there shouldn't be any issues. Do you have more details about what's going wrong?

**Comment by TysonMN** [comment]
> In theory, as long as you have LFS installed, there shouldn't be any issues.

That was the fix.  All I had to do was install it (from [here](https://git-lfs.github.com/)).

Thanks! :)

**Comment by RehanSaeed** [comment]
Cool!

FYI, this blog post I wrote might be helpful:

https://rehansaeed.com/gitattributes-best-practices/
